src/Ssh/Ssh/ChangeLog.md

```diff
@@ -19,6 +19,10 @@
 -->
 
 ## Upcoming Release
+## Version 0.1.2
```

Comment:
The version number is calculated by tool. Please don't add version here next time.